### ISSUE TITLE
[PostgreSQL] Salesforce integration. Error on update to v3

### ISSUE THREAD

**Issue by umkasanki** [verbb/formie#2094]
### Describe the bug

Updating the project to CraftCMS 5 with PostgreSQL .
When applying migrations, the following error occurs:
```
...
*** applying m231129_000000_integrations_mapping
    > update in {{%formie_forms}} ... done (time: 0.012s)
    > update in {{%formie_forms}} ... done (time: 0.001s)
    > update in {{%formie_forms}} ... done (time: 0.002s)
    > update in {{%formie_forms}} ... done (time: 0.001s)
    > update in {{%formie_forms}} ... done (time: 0.001s)
    > update in {{%formie_forms}} ... done (time: 0.001s)
    > update in {{%formie_forms}} ... done (time: 0.001s)
    > update in {{%formie_forms}} ... done (time: 0.002s)
    > update in {{%formie_forms}} ... done (time: 0.002s)
    > update in {{%formie_forms}} ... done (time: 0.002s)
    > update in {{%formie_forms}} ... done (time: 0.001s)
    > update in {{%formie_forms}} ... done (time: 0.002s)
    > update in {{%formie_forms}} ... done (time: 0.002s)
    > update in {{%formie_forms}} ... done (time: 0.001s)
    > update in {{%formie_forms}} ... done (time: 0.001s)
*** applied m231129_000000_integrations_mapping (time: 0.064s)

*** applying m231130_000000_conditions_mapping
*** applied m231130_000000_conditions_mapping (time: 0.016s)

*** applying m231202_000000_auth_module
    > create table {{%auth_oauth_tokens}} ... done (time: 0.017s)
Exception: SQLSTATE[42883]: Undefined function: 7 ERROR:  operator does not exist: character varying = integer
LINE 3: ...orce') AND ("tokenType"='oauth2') AND ("reference"=2) AND ("...
                                                             ^
HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.
The SQL being executed was: SELECT *
FROM "auth_oauth_tokens"
WHERE ("ownerHandle"='formie') AND ("providerType"='verbb\formie\integrations\crm\Salesforce') AND ("tokenType"='oauth2') AND ("reference"=2) AND ("accessToken"='00D7Q000000KCYW!AQEAQD2FLyIod9JUYM2y4rgPe4oe0Ve8I5VZneJhTzG2puBEh3FuKqsuFJ6EhgyF4T_pIRIJ2_qC3XeWBTSHsFiXvYiSPsiw') AND ("secret" IS NULL) AND ("expires" IS NULL) AND ("refreshToken"='5Aep861yNl8cSWS3xVYm8.5PXxYmtGOByDgt8j_o7M24Net28R_LWkMv2XbleAzjiw9Ma.SOGuQu2Rx5zk7kP9d') AND ("dateCreated"='2023-05-03 15:13:42') AND ("dateUpdated"='2024-10-17 10:43:17') AND ("uid"='b78af6ab-fec0-47ca-b495-d2dbe65001e4') (/app/vendor/yiisoft/yii2/db/Schema.php:676)
#0 /app/vendor/yiisoft/yii2/db/Command.php(1325): yii\db\Schema->convertException(Object(PDOException), 'SELECT *\nFROM "...')
#1 /app/vendor/yiisoft/yii2/db/Command.php(1186): yii\db\Command->internalExecute('SELECT *\nFROM "...')
#2 /app/vendor/yiisoft/yii2/db/Command.php(417): yii\db\Command->queryInternal('fetchAll', NULL)
#3 /app/vendor/yiisoft/yii2/db/Query.php(249): yii\db\Command->queryAll()
#4 /app/vendor/craftcms/cms/src/db/Query.php(292): yii\db\Query->all(NULL)
#5 /app/vendor/verbb/formie/src/migrations/m231202_000000_auth_module.php(58): craft\db\Query->all()
#6 /app/vendor/craftcms/cms/src/db/Migration.php(50): verbb\formie\migrations\m231202_000000_auth_module->safeUp()
#7 /app/vendor/yiisoft/yii2/console/controllers/BaseMigrateController.php(758): craft\db\Migration->up()
#8 /app/vendor/craftcms/cms/src/console/controllers/MigrateController.php(382): yii\console\controllers\BaseMigrateController->migrateUp('m231202_000000_...')
#9 [internal function]: craft\console\controllers\MigrateController->actionAll()
#10 /app/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#11 /app/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#12 /app/vendor/yiisoft/yii2/console/Controller.php(180): yii\base\Controller->runAction('all', Array)
#13 /app/vendor/craftcms/cms/src/console/controllers/MigrateController.php(195): yii\console\Controller->runAction('all', Array)
#14 /app/vendor/yiisoft/yii2/base/Module.php(552): craft\console\controllers\MigrateController->runAction('all', Array)
#15 /app/vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction('migrate/all', Array)
#16 /app/vendor/craftcms/cms/src/console/Application.php(91): yii\console\Application->runAction('migrate/all', Array)
#17 /app/vendor/yiisoft/yii2/base/Controller.php(212): craft\console\Application->runAction('migrate/all', Array)
#18 /app/vendor/craftcms/cms/src/console/Controller.php(207): yii\base\Controller->run('migrate/all', Array)
#19 /app/vendor/craftcms/cms/src/console/controllers/UpController.php(60): craft\console\Controller->run('migrate/all', Array)
#20 [internal function]: craft\console\controllers\UpController->actionIndex()
#21 /app/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#22 /app/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#23 /app/vendor/yiisoft/yii2/console/Controller.php(180): yii\base\Controller->runAction('', Array)
#24 /app/vendor/craftcms/cms/src/console/ControllerTrait.php(88): yii\console\Controller->runAction('', Array)
#25 /app/vendor/craftcms/cms/src/console/Controller.php(216): craft\console\Controller->traitRunAction('', Array)
#26 /app/vendor/yiisoft/yii2/base/Module.php(552): craft\console\Controller->runAction('', Array)
#27 /app/vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction('up', Array)
#28 /app/vendor/craftcms/cms/src/console/Application.php(91): yii\console\Application->runAction('up', Array)
#29 /app/vendor/yiisoft/yii2/console/Application.php(147): craft\console\Application->runAction('up', Array)
#30 /app/vendor/craftcms/cms/src/console/Application.php(122): yii\console\Application->handleRequest(Object(craft\console\Request))
#31 /app/vendor/yiisoft/yii2/base/Application.php(384): craft\console\Application->handleRequest(Object(craft\console\Request))
#32 /app/craft(23): yii\base\Application->run()
#33 {main}
*** failed to apply m231202_000000_auth_module (time: 0.078s)


35 from 47 migrations were applied.

Migration failed. The rest of the migrations are canceled.

Restore a database backup before trying again.

Aborting remaining tasks.
```

### Steps to reproduce

1. Run Craft 5 update

### Form settings

- Multi-page form: No
- Submission Method: Both
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

Craft Pro 4.12.6

### Plugin version

2.1.31

### Multi-site?

No

### Additional context

Used Formie Salesforce integration.
PostgreSQL 14.13

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.8"`.

**Comment by engram-design** [comment]
Fixed in [3.0.9](https://github.com/verbb/formie/releases/tag/3.0.9)
